pom.xml

```diff
@@ -60,6 +62,7 @@ THE SOFTWARE.
     <maven3.0.x.version>3.0.5</maven3.0.x.version>
     <maven3.1.x.version>3.1.0</maven3.1.x.version>
     <maven3.2.x.version>3.2.3</maven3.2.x.version>
+    <maven3.3.x.version>3.3.1</maven3.3.x.version>
```

Comment:
I'm not sure if we should use the lowest or the highest version. For others versions it's nor the lowest or the highest version ... cc @olamy